problem-12.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+// Takes a number as input and determines whether it is an Armstrong number or not
+
+int exponent(int number, int power)

```

Comment:
It's good that you have a separate function for this, generally it's a good practice to document functions like this

```c
/**
 * Adds two integers and returns their sum.
 *
 * This function takes two integers as input, adds them together, and returns
 * the result as an integer. It performs basic addition and does not handle
 * overflow or error cases.
 *
 * @param num1 The first integer to be added.
 * @param num2 The second integer to be added.
 *
 * @return The sum of num1 and num2.
 */
int addTwoNumbers(int num1, int num2) {
    return num1 + num2;
}
```

Reply:
Only add description if you think it adds value.